lib/client-assets.php

```diff
@@ -205,6 +205,7 @@ function gutenberg_register_scripts_and_styles() {
 				'	wp.data',
 				'		.use( wp.data.plugins.persistence, { storageKey: storageKey } )',
 				'		.use( wp.data.plugins.controls );',
+				'	wp.data.plugins.persistence.__unstableMigrate( { storageKey: storageKey } );',
```

Comment:
As part of reconciling #13569, it occurs to me we'll need some plan for how to bring this into core (e.g. Trac ticket timed with introduction of block editor module to core).